{"sections":[],"kind":"symbol","abstract":[{"type":"text","text":"The reason why a background URLSessionTask was cancelled."}],"schemaVersion":{"major":0,"patch":0,"minor":3},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError\/backgroundTaskCancelledReason-swift.property"},"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["macOS"],"languages":["swift"],"tokens":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"backgroundTaskCancelledReason","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:10Foundation8URLErrorV","text":"URLError","identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError","kind":"typeIdentifier"},{"kind":"text","text":"."},{"text":"BackgroundTaskCancelledReason","kind":"typeIdentifier","preciseIdentifier":"s:10Foundation8URLErrorV29BackgroundTaskCancelledReasonO","identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError\/BackgroundTaskCancelledReason-swift.enum"},{"text":"? { ","kind":"text"},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}]}]}],"hierarchy":{"paths":[["doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX","doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError"]]},"metadata":{"fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"backgroundTaskCancelledReason","kind":"identifier"},{"kind":"text","text":": "},{"text":"URLError","kind":"typeIdentifier","preciseIdentifier":"s:10Foundation8URLErrorV"},{"text":".","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:10Foundation8URLErrorV29BackgroundTaskCancelledReasonO","text":"BackgroundTaskCancelledReason"},{"text":"?","kind":"text"}],"title":"backgroundTaskCancelledReason","extendedModule":"Foundation","modules":[{"name":"FoundationX","relatedModules":["Foundation"]}],"roleHeading":"Instance Property","externalID":"s:10Foundation8URLErrorV29backgroundTaskCancelledReasonAC010BackgrounddeF0OSgvp","role":"symbol","platforms":[{"introducedAt":"13.0","deprecated":false,"name":"iOS","unavailable":false,"beta":false},{"name":"macOS","unavailable":false,"beta":false,"deprecated":false,"introducedAt":"10.15"},{"deprecated":false,"introducedAt":"13.0","unavailable":false,"name":"tvOS","beta":false},{"introducedAt":"6.0","beta":false,"name":"watchOS","unavailable":false,"deprecated":false}],"symbolKind":"property"},"variants":[{"paths":["\/documentation\/foundationx\/urlerror\/backgroundtaskcancelledreason-swift.property"],"traits":[{"interfaceLanguage":"swift"}]}],"references":{"doc://com.swallow.documentation.FoundationX/documentation/FoundationX/URLError/backgroundTaskCancelledReason-swift.property":{"kind":"symbol","url":"\/documentation\/foundationx\/urlerror\/backgroundtaskcancelledreason-swift.property","type":"topic","role":"symbol","title":"backgroundTaskCancelledReason","abstract":[{"text":"The reason why a background URLSessionTask was cancelled.","type":"text"}],"fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"backgroundTaskCancelledReason","kind":"identifier"},{"kind":"text","text":": "},{"text":"URLError","kind":"typeIdentifier","preciseIdentifier":"s:10Foundation8URLErrorV"},{"kind":"text","text":"."},{"preciseIdentifier":"s:10Foundation8URLErrorV29BackgroundTaskCancelledReasonO","kind":"typeIdentifier","text":"BackgroundTaskCancelledReason"},{"text":"?","kind":"text"}],"identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError\/backgroundTaskCancelledReason-swift.property"},"doc://com.swallow.documentation.FoundationX/documentation/FoundationX":{"abstract":[],"kind":"symbol","title":"FoundationX","identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX","url":"\/documentation\/foundationx","role":"collection","type":"topic"},"doc://com.swallow.documentation.FoundationX/documentation/FoundationX/URLError":{"title":"URLError","role":"symbol","url":"\/documentation\/foundationx\/urlerror","type":"topic","kind":"symbol","identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError","abstract":[{"text":"Describes errors in the URL error domain.","type":"text"}],"navigatorTitle":[{"text":"URLError","kind":"identifier"}],"fragments":[{"text":"struct","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"URLError"}]},"doc://com.swallow.documentation.FoundationX/documentation/FoundationX/URLError/BackgroundTaskCancelledReason-swift.enum":{"title":"URLError.BackgroundTaskCancelledReason","abstract":[{"text":"Reasons used by URLError to indicate why a background URLSessionTask was cancelled.","type":"text"}],"fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"kind":"identifier","text":"BackgroundTaskCancelledReason"}],"role":"symbol","type":"topic","url":"\/documentation\/foundationx\/urlerror\/backgroundtaskcancelledreason-swift.enum","navigatorTitle":[{"text":"BackgroundTaskCancelledReason","kind":"identifier"}],"kind":"symbol","identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError\/BackgroundTaskCancelledReason-swift.enum"}}}